src/component/cards.test.js

```diff
@@ -18,3 +32,13 @@ test('should be able to search for a movie', async () => {
   const matching = resp.every(handleTitle);
   expect(matching).toBe(false);
 });
+
+test('Should search for a movie by Title', async () => {
+  render(<App />);
+
+  const searchBar = await screen.findByRole('textbox');
+
+  const characterName = 'Cowboy Bebop';
+
+  userEvent.type(searchBar, characterName);
```

Comment:
Would be a good place to add an assertion that checks the outcome of the user event